src/main/resources/reference.conf

```diff
@@ -41,5 +47,13 @@ kamon {
     # to scale before sending to datadog set "memory-units" to "gb" or "mb" or "kb".
     # Value "b" is equivalent to omitting the setting
     information-unit = "b"
+
+    tag-filters {
+      includes = [ ".*" ]
+      excludes = []
+    }
+
+    service-tag-name = "service"
```

Comment:
We just merged a tool that will help getting all environment-related tags configured easily, please take a look at this: https://github.com/kamon-io/Kamon/pull/527.

I'll publish a minor release with that utility later today so that we can move this code to use that, wdyt?